lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -79,7 +80,7 @@ class VerificationDataSerializer(serializers.Serializer):
     status_date = serializers.DateTimeField()
 
 
-class ProgressTabSerializer(serializers.Serializer):
+class ProgressTabSerializer(VerifiedModeSerializerMixin, serializers.Serializer):
```

Comment:
Should be good to remove the serializer.Serializer because the mixin inherits from that already.